cdap-data-fabric/src/main/java/co/cask/cdap/data/file/FileWriter.java

```diff
@@ -42,4 +42,5 @@
    * @throws IOException if fail to append
    */
   void appendAll(Iterator<? extends T> events) throws IOException;
+
```

Comment:
its leftover from when I removed the method from this interface. I'll remove it.